### PR TITLE
Gp/bootnodes chain spec

### PR DESCRIPTION
**Pull request by JackPiri**
Modifications for chain spec (testnet only):

- added bootnodes connection strings
- added telemetry endpoint

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -184,6 +185,18 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
...
+    .with_boot_nodes(vec![
+        "/dns/nh-bootnode-1.horizenlabs.io/tcp/443/wss/p2p/12D3KooWNhvf6iSowraUY4tZnjpNZXEe85oy9zDWYRKFBnWivukc"
+        .parse()
+        .expect("MultiaddrWithPeerId"),
+        "/dns/nh-bootnode-2.horizenlabs.io/tcp/443/wss/p2p/12D3KooWEjVadU1YWyfDGvyRXPbCq2rXhzJtXaG4RxJZBkGE9Aug"
+        .parse()
+        .expect("MultiaddrWithPeerId"),
+    ])
+    .with_telemetry_endpoints(
+		TelemetryEndpoints::new(vec![("wss://nh-telemetry.horizenlabs.io/submit".to_string(), 1)])
```
Comment (drgora) [inline node/src/chain_spec.rs:197]: I'd suggest replacing the verbosity level value with its named constant.
```suggestion
		TelemetryEndpoints::new(vec![("wss://nh-telemetry.horizenlabs.io/submit".to_string(), sc_telemetry::CONSENSUS_INFO)])
```

**Review by drgora — COMMENTED** [review]
Another note: I believe we should either remove the `STAGING_TELEMETRY_URL` (currently commented out), or actually use it with our endpoint in place of the raw value at line 197.

**Review by la10736 — APPROVED** [review]
LGTM

File: node/src/chain_spec.rs (hunk 1 above)
Comment (JackPiri) [reply node/src/chain_spec.rs]: Ok I agree on both notes.
I have replaced `1` with `sc_telemetry::CONSENSUS_INFO` and now `STAGING_TELEMETRY_URL` is set and used (done similarly also for bootnodes connection strings).

**Review by drgora — APPROVED** [review]
Looks good to me

**Review by lander86 — APPROVED** [review]
LGTM

**Pull request merged**